pkg/steps/source.go

```diff
@@ -37,22 +38,29 @@ const (
 
 	ProwJobIdLabel = "prow.k8s.io/id"
 
-	gopath = "/go"
+	gopath     = "/go"
+	sshKeyPath = "/sshprivatekey"
 )
 
 var (
 	JobSpecAnnotation = fmt.Sprintf("%s/%s", CiAnnotationPrefix, "job-spec")
 )
 
-func sourceDockerfile(fromTag api.PipelineImageStreamTagReference, workingDir string) string {
-	return fmt.Sprintf(`
-FROM %s:%s
```

Comment:
Would prefer not to go away from this string formatting

Reply:
yea the difference is only an `\n` both at end and beginning. I will address this and not make changes to the tests too.